scripts/inference.py

```diff
@@ -89,7 +89,7 @@
     device = torch.device(args.device_type)
 
 torch.set_default_device(device)
-torch.set_default_dtype(torch.half)
+torch.set_default_dtype(torch.bfloat16)
```

Comment:
Why was this required?

Reply:
oh, accidentally adding this file, will revert it

Reply:
fixed